### PR TITLE
fix(paths): resolve issues with paths modal and start point selector

### LINKED ISSUE #9407: Not able to explore users in path flow "Load people on reducer People failed: A server error occurred."
## Bug description

When I click on the no. of users reported for drop off in any path, it is supposed to show me the list of users. 
But this throws a server error:

image
2022-04-13 17_17_39-TradesViz Stats • Dashboards • PostHog
2022-04-13 17_17_46-TradesViz Stats • Dashboards • PostHog

## How to reproduce

1. Click on any path and then click on the no. of users

## Environment

- [x] PostHog Cloud

## Additional context

This started occurring only recently.

### PR DESCRIPTION
**Pull request by neilkakkar**
## Problem

Fixes https://github.com/PostHog/posthog/issues/9407

[More context here](https://posthogusers.slack.com/archives/C01GLBKHKQT/p1649840258080379) (users group slack thread)

## Changes

The modal was only being loaded on the pathsTab, fixes this to load it on the main Paths insight instead.

Also start point was annoyingly jittery: select one, and the results would vanish, because the count wasn't being computed correctly.

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Run locally, go to a paths insight, see modal works both outside and inside edit mode.

Then, try inputting a start point, see it works.

### CODE REVIEW

File: frontend/src/scenes/paths/Paths.tsx [hunk 1]
```diff
@@ -264,240 +269,260 @@ export function Paths(): JSX.Element {
...
+            <PersonsModal
+                visible={showingPeople && !cohortModalVisible}
+                view={InsightType.PATHS}
+                filters={filter}
+                onSaveCohort={() => {
+                    setCohortModalVisible(true)
+                }}
+                aggregationTargetLabel={{ singular: 'user', plural: 'users' }}
+            />
+            <div className="paths-container" id={`'${insight?.short_id || DEFAULT_PATHS_ID}'`}>
```
Comment (neilkakkar) [inline frontend/src/scenes/paths/Paths.tsx:282]: just whitespace change

File: frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts [hunk 2]
```diff
@@ -157,7 +157,10 @@ export const infiniteListLogic = kea<infiniteListLogicType>({
                         ),
                         searchQuery: values.searchQuery,
                         queryChanged,
-                        count: response.count || 0,
+                        count:
+                            response.count ||
+                            (Array.isArray(response) ? response.length : 0) ||
```
Comment (liyiy) [inline frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts:162]: When does `response.count` not cover the result we need? Just for paths results?

Comment (neilkakkar) [reply frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts:162]: for paths, and everything that's about to come with universal search, those have different response structures too.

Comment (neilkakkar) [reply frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts:162]: https://github.com/PostHog/posthog/pull/9299/files#diff-b25cfa2d4b96abd40174cb762dab262471bcb6216e33e86d49c9d612d963cc67R173

**Review by liyiy — APPROVED** [review]

**Pull request merged**